TombEngine/Game/control/control.cpp

```diff
@@ -94,6 +96,8 @@ short NextItemFree;
 short NextFxActive;
 short NextFxFree;
 
+int ControlPhaseTime;
```

Comment:
Please remove debug values or move it to another file. Preferably, we need to have separate `debug.cpp/.h` files from which you will call frame time counters.

Reply:
Yeah it's not optimal. But in the next release I would implement more timings from control phase and so your suggestion is valid.